GPCS4/SceModules/SceLibkernel/sce_libkernel.cpp

```diff
@@ -47,6 +49,40 @@ int PS4API __pthread_cxa_finalize(void)
 }
 
 
+int PS4API scek__write(int fd, const void* buf, size_t size)
+{
+	if (fd > 2)
+	{
+		LOG_FIXME("Not implemented for non-stdout");
```

Comment:
I don't like it, use `write()` for given fd, it opened by host earlier

Reply:
Ah I thought that wasn't implemented